java/picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java

```diff
@@ -81,44 +86,44 @@ protected void onCreate(Bundle savedInstanceState) {
     //[DocRef: Name=Picture Marker Symbol URL, Category=Fundamentals, Topic=Symbols and Renderers]
     //Create a picture marker symbol from a URL resource
     //When using a URL, you need to call load to fetch the remote resource
-    final PictureMarkerSymbol campsiteSymbol = new PictureMarkerSymbol(
+    mCampsiteSymbol = new PictureMarkerSymbol(
         "http://sampleserver6.arcgisonline"
             + ".com/arcgis/rest/services/Recreation/FeatureServer/0/images/e82f744ebb069bb35b234b3fea46deae");
     //Optionally set the size, if not set the image will be auto sized based on its size in pixels,
     //its appearance would then differ across devices with different resolutions.
-    campsiteSymbol.setHeight(18);
-    campsiteSymbol.setWidth(18);
-    campsiteSymbol.loadAsync();
+    mCampsiteSymbol.setHeight(18);
+    mCampsiteSymbol.setWidth(18);
+    mCampsiteSymbol.loadAsync();
     //[DocRef: END]
-    campsiteSymbol.addDoneLoadingListener(new Runnable() {
+    mCampsiteSymbol.addDoneLoadingListener(new Runnable() {
```

Comment:
as above, I think we can rely on loadable chain here and therefore avoid another global property

Reply:
Yes, it looks like a lot of the samples do the pattern you have here i.e. load something then when loading is done add it to something that would have loaded it automatically anyway. It is worth remembering you can still have the done loading listeners to catch an error even if you use automatic loading.